examples/otlp/start.py

```diff
@@ -0,0 +1,19 @@
+import time
+from autometrics import autometrics, init
+
+init(
+    exporter={
```

Comment:
so i needed to fill in all fields like this to get things working (related to my comment about providing defaults for the exporter config):

```python
    exporter={
        "type": "otlp-proto-http",
        "endpoint": "http://localhost:4318/v1/metrics",
        "insecure": True,
        "headers": None,
        "credentials": None,
        "push_interval": 1000,
        "timeout": 1000,
        "preferred_temporality": None,
    },
```